examples/rest/calendar_recurrence.py

```diff
@@ -1,15 +1,150 @@
-from dynamicforms import serializers
+from typing import List
+
+from django.utils.translation import gettext_lazy as _
+from rest_framework.exceptions import ValidationError
+
+from dynamicforms import fields, serializers
+from dynamicforms.mixins import F
 from examples.enum_field import EnumField
 from examples.models import CalendarRecurrence
 
 
+class DaysField(fields.CharField):
+    def to_internal_value(self, data: str):
+        def parse_token(token):
+            token = token.strip()
+            if len(token.split(" ")) == 2:
+                return token.split(" ")
+            return token
+
+        if len(data.split(",")):
+            return [parse_token(token) for token in data.split(",")]
+        return parse_token(data)
+
+    def to_representation(self, value: list, row_data=None):
+
+        return ", ".join(value)
+
+
+class DaysValidator:
+    message = _("Wrong input format")
+
+    def __init__(self, message=None):
+        self.message = message or self.message
+
+    def __call__(self, value: list):
+        def token_validation(token: str):
+            if token is None:
+                return
+            token = token.strip()
+            if len(token.split(" ")) == 2:
+                d = token.split(" ")
+                if d[0][:3].lower() not in (
+                    "1st",
+                    "fir",
+                    "2nd",
+                    "sec",
+                    "3rd",
+                    "thi",
+                    "4th",
+                    "fou",
+                    "5th",
+                    "fif",
+                    "las",
+                ) or d[1] not in ("mo", "tu", "we", "th", "fr", "sa", "su"):
```

Comment:
Tukajle bi bilo skoraj bolje, če bi vlekel iz django spremenljivk, ker ima Djago to že v locale pluginu take konstante...